Formula/cockroach.rb

```diff
@@ -1,19 +1,17 @@
 class Cockroach < Formula
   desc "Distributed SQL database"
   homepage "https://www.cockroachlabs.com"
-  version "1.0-rc.1"
-  url "https://binaries.cockroachdb.com/cockroach-v1.0-rc.1.src.tgz"
-  sha256 "162e3954ff41515bf2ab2017ab38f9cdf76775d7d67fdeead343234d470dca6c"
+  version "1.0-rc.2"
+  url "https://binaries.cockroachdb.com/cockroach-v1.0-rc.2.src.tgz"
+  sha256 "f4b2a32748427b7648f589214e24a5ffc40719e56cb97859c859bf0009ddd543"
   head "https://github.com/cockroachdb/cockroach.git"
 
   depends_on "cmake" => :build
   depends_on "go" => :build
   depends_on "xz" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    system "make", "install"
-    bin.install "bin/cockroach" => "cockroach"
+    system "make", "install", "prefix=" + prefix
```

Comment:
if y'all are making more changes, care to appease me with "prefix=${prefix}"? thanks!